packages/next/client/link.tsx

```diff
@@ -99,7 +99,11 @@ const listenToIntersections = (el: any, cb: any) => {
   observer.observe(el)
   listeners.set(el, cb)
   return () => {
-    observer.unobserve(el)
+    try {
+      observer.unobserve(el)
+    } catch (_) {
+      /* not detrimental if it fails to unobserve */
```

Comment:
I think this should be `console.error(err)`

Reply:
It seems to be a browser bug so should we log it out even though the user can't address it?

Reply:
Yes, otherwise it will completely disappear.